### PR TITLE
Add PoliCTF 2015 Skeleton

### PR DESCRIPTION
**Pull request by fariskhi**
:books: Add PoliCTF 2015 Skeleton

### CODE REVIEW

**Comment by dhanvi** [comment]
Well I was just thinking of doing the same if @YASME-Tim is not doing it :smile: 

also you need to keep the every file in the repo not in their server they might put it down after some time :smile:

File: polictf-2015/README.md [hunk 1]
```diff
@@ -0,0 +1,41 @@
+# ASIS Quals CTF 2015 write-ups
```
Comment (dhanvi) [inline polictf-2015/README.md]: well it's polictf no probs I will fix them soon.

File: polictf-2015/crypto/3doges2/README.md [hunk 2]
```diff
@@ -0,0 +1,19 @@
...
+**Category:** Crypto
+**Points:** 400
+**Solves:** 29
+**Description:**
+
+> Wanna talk with John? Join his supersecret communication channel. Of course it's secure, it uses a brand new hipster handmade cryptocipher! Amazingly unbreakable! The party is on
+> thedoges.polictf.it:80
+> GPG key: oliphushIkDeedTarWodVenEfyockOc2
+> [http://download.polictf.it/3doges2_73e23df92b0305a9d90c955c7f4ca7a2.tar.gz.gpg](Download)
+
```
Comment (dhanvi) [inline polictf-2015/crypto/3doges2/README.md]: as I said in previous comment we need the file locally

File: polictf-2015/crypto/exorcise/README.md [hunk 3]
```diff
@@ -0,0 +1,17 @@
+# PoliCTF 2015: Even the server lies (on a couch unfortunatel
```
Comment (dhanvi) [inline polictf-2015/crypto/exorcise/README.md]: the name should have been exorcise I guess

File: polictf-2015/crypto/prophet-said/README.md [hunk 4]
```diff
@@ -0,0 +1,18 @@
+# PoliCTF 2015: Even the server lies (on a couch unfortunatel
```
Comment (dhanvi) [inline polictf-2015/crypto/prophet-said/README.md]: same here as prophet-said

**Comment by dhanvi** [comment]
also for the web challenges we need to add all the files of web for reference (you can use wget -r )

**Comment by fariskhi** [comment]
I'm sorry for my careless. Thanks for fixing it :smiley: 
I will add the copy of all files later

**Comment by dhanvi** [comment]
its ok I am adding the files right now :smile: 

Thanks again for adding them

**Comment by dhanvi** [comment]
also we have some files which are available only via torrents (files which are very big) 

we cann't add them to the repo as they will make the repo slow so we should add them somewhere in the cloud

also I can't access p2p protocol as I am under a strict firewall :(  hope you can add them to the cloud and mention the link here or we can raise a new issue here.

reversing/am i cheating

**Pull request merged**